IPython/core/oinspect.py

```diff
@@ -694,7 +694,8 @@ def pinfo(self, obj, oname='', formatter=None, info=None, detail_level=0):
     def info(self, obj, oname='', formatter=None, info=None, detail_level=0):
         """DEPRECATED. Compute a dict with detailed information about an object.
         """
-        warnings.warn('Inspector.info is deprecated as of IPython 5.0',
+        if formatter is not None:
+            warnings.warn('Inspector.info(formatter) keyword is deprecated as of IPython 5.0 and will have no effects.',
```

Comment:
I did not understand the wording right away. What about
The `formatter` keyword argument to `inspector.info` is deprecated as of IPython 5.0 and will have no effects.